go/pkg/yolotree/yolotree.go

```diff
@@ -0,0 +1,22 @@
+package yolotree
+
+import (
+	"berty.tech/yolo/v2/go/pkg/yolopb"
+	"fmt"
+	"sort"
```

Comment:
```suggestion
	"berty.tech/yolo/v2/go/pkg/yolopb"
	"fmt"
	"sort"
```

Your editor won't sort imports, you should config it.

Reply:
done :)